### PR TITLE
Removes the unintended ability to have sex with cats

### LINKED ISSUE #7827: You can procreate with pets
1. Get polymorphed into an animal
2. Find the matching station pet of the opposing gender
3. Find a quiet place away from other people
4. Log out
5. Log in
6. Never feel clean again

:cat2: :cat2: :cat2: :cat2: :cat2:

### PR DESCRIPTION
**Pull request by Incoming5643**
Veni, Vidi, Futuō

:cat2: :cat2: :cat2: :cat2: :cat2: 
fixes #7827

### CODE REVIEW

**Comment by Incoming5643** [comment]
Does NOT need a changelog

**Comment by Cheridan** [comment]
>DEPRIVING THE PLAYERBASE OF THE JOY OF CHILDBIRTH

**Comment by GunHog** [comment]
You are, and forever will be, a monster.

**Comment by ike709** [comment]
time to abuse it before this is merged

**Comment by Dorsisdwarf** [comment]
This needs the changelog "Removes ERP mechanics".

That'll confuse 'em.

**Comment by Niknakflak** [comment]
This is hilarious

**Comment by MrStonedOne** [comment]
Can we please stop fucking titleing these fix prs so shittyly?

How about giving it an actual title, that states what it actually fixes, rather than giving up some random number that means nothing to 90% of the people reading the PR list

**Comment by Incoming5643** [comment]
You asked for it

File: code/modules/mob/living/simple_animal/simple_animal.dm [hunk 1]
```diff
@@ -462,7 +462,7 @@
 		else if(istype(M, childtype)) //Check for children FIRST.
 			children++
 		else if(istype(M, species))
-			if(M.client)
+			if(M.ckey)
 				continue
 			else if(!istype(M, childtype) && M.gender == MALE) //Better safe than sorry ;_;
```
Comment (Miauw62) [inline code/modules/mob/living/simple_animal/simple_animal.dm]: >better safe than sorry

**Comment by Cheridan** [comment]
time to codecuck

**Pull request merged**